melodic-manatees/early_internet/background_app/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
There's no point in this file existing if you don't have any tests. It would be better to just delete it, because right now it's just bloat in your PR.